src/coreclr/jit/emitriscv64.cpp

```diff
@@ -2110,19 +2110,6 @@ void emitter::emitJumpDistBind()
 #endif // DEBUG
 }
 
-/*****************************************************************************
- *
- *  Emit a 32-bit RISCV64 instruction
- */
-
-unsigned emitter::emitOutput_Instr(BYTE* dst, code_t code) const
-{
-    assert(dst != nullptr);
-    assert(sizeof(code_t) == 4);
-    memcpy(dst + writeableOffset, &code, sizeof(code_t));
-    return sizeof(code_t);
-}
```

Comment:
One reason was name confusion, we had emitOutputInstr and emitOutput_Instr which did different things under almost the same name. Another is to reuse common code rather than maintaining our own helpers that do the same. Do you see any problem with emitOutputLong for RISC-V?

Reply:
Thank you for the comment.
- I think `emitOutput_Instr` is more intuitive than `emitOutputLong`. Actually, I thought `emitOutputLong` writes long value (8 bytes) into dst.
- ARM64 and Loongarch64 use `emitOutput_Instr`. XARCH use `emitOutputLong` and others such as `emitOutputByte`, `emitOutputWord` and so on. I want RISC-V looks similar to ARM64 and LoongArch64 than the others because RISC-V referenced many things from ARM64 and LoongArch64.

Reply:
You're right, only xarch uses it. OK let's keep emitOutput_Instr. Thank you.

Reply:
How do you think about changing implementation from 'memcpy' to storing value to the pointer like arm64?
I think the function will be called very many and 'memcpy' overhead is bigger than storing. I think such overhead is unnecessary.

Reply:
No problem

Reply:
Thank you

Reply:
As side note/nit: when N is very small and known at compile time usually memcpy(N)/memset(N) are lowered by 
Clang/GCC (-O2/-O3) to equivalent loads/stores so it's likely that both forms are fine (at least on Linux :).

Reply:
I know but:
* it's lowered to a sequence of sb + srli vs. a single sw
* no need to pay the tax in debug mode